### PR TITLE
fixed API endpoint for search podcast (fix #241)

### LINKED ISSUE #241: Searching for long URL causes server error
Error report

````
Internal Server Error: /search.json

DataError at /search.json
value too long for type character varying(2048)


Request Method: GET
Request URL: https://www.gpodder.net/search.json?q=http://www.masterads.info/instagram/campanha.php?id=lNmb19mbuFmZyUSO2kjNhNTJnJ3buIXZrNWYyRXLuJ3bjB3bw5yYpxmY1BnZyUiZyUSYzUCckVXPyR3Ow1WYmU2YuV3bu5WYmJTJwEzMzE2Ml02bj5CdzMDau02bkNzMyZmZyUiZyUSYzUCckVXPyR3Ow1WYmU2YuV3bu5WYmJTJwgTYzUSZk5yYjNmLyV2ajFmc0ZmMlYmMlE2MlAHZ11jc0tDctFmJlNmb19mbuFmZyUSbvNmLnJmchJnLxEjZyUiZyUSYzUCckVXPyR3Ow1WYmU2YuV3bu5WYmJTJ3MzMxE2Mlcmcv5icrNWYyRnblB3buIXZrNWYyRnZyUiZyUSYzUCckVXPyR3Ow1WYmU2YuV3bu5WYmJTJwgTYzUSbvNmL0J2YpxmY1BnLyV2ajFmc0ZmMlYmMlE2MlAHZ11jc0tDctFmJlNmb19mbuFmZyUCM4E2MlUWbucmYyFmcuITMmJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMlADOhNTJ1xmL5JnchhmLyV2ajFmc05CN2BXamJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMlADOhNTJhVnL4VmLyV2ajFmc0ZmMlYmMlE2MlAHZ11jc0tDctFmJlNmb19mbuFmZyUCM0cjMhNTJt92YucmYyFmcukjZyUiZyUSYzUCckVXPyR3Ow1WYmU2YuV3bu5WYmJTJwgTYzUyZy9mL4dzMzEjLyV2ajFmc0ZmMlYmMlE2MlAHZ11jc0tDctFmJlNmb19mbuFmZyUyb05ScyBnLyV2ajFmc0ZmMlYmMlE2MlAHZ11jc0tDctFmJlNmb19mbuFmZyUSbvNmLzRnblJncvRXL5ZWa55iclt2YhJHdmJTJmJTJhNTJwRHdo1jc0tDctFmJlNmb19mbuFmZyUCM4E2Ml02bj5CduVmcy9Gd0lmYuVGcv5iclt2YhJHdmJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMlUGZucmbppXYsJmLyV2ajFmc0ZmMlYmMlE2MlAHd0hWPyR3Ow1WYmU2YuV3bu5WYmJTJwgTYzUCdp5SZs9GdzlmLyV2ajFmc0ZmMlYmMlE2MlAHZ11jc0tDctFmJlNmb19mbuFmZyUCM4E2Ml02bj5CdiNWasJWdw5iclt2YhJHdmJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMlkjN5YTYzUSbvNmLj5WezVGZuMXdk9GelZmMlYmMlE2MlAHd0hWPyR3Ow1WYmU2YuV3bu5WYmJTJ5YTO2E2Ml42Yu02bj5CZuFGblJXYj5CdiZmMlYmMlE2MlAHd0hWPyR3Ow1WYmU2YuV3bu5WYmJTJ5YTO2E2MlsGduIXZmJXdzJXZwB3bj5iclt2YhJHdmJTJmJTJhNTJwRHdo1jc0tDctFmJlNmb19mbuFmZyUSbvNmLj5WezVGZuMXdk9GelZmMlYmMlE2MlAHd0hWPyR3Ow1WYmU2YuV3bu5WYmJTJwEzMzE2Ml02bj5CdzMDauc3d3ZmMlYmMlE2MlAHd0hWPyR3Ow1WYmU2YuV3bu5WYmJTJwgTYzUSbvNmLzRnblJncvRXL5ZWa55iclt2YhJHdmJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMl02bj5yN39GcuIDdmJTJmJTJhNTJwRHdo1jc0tDctFmJlNmb19mbuFmZyUSO2kjNhNTJrRnLyVmZyV3cyVGcw92YmJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMlkjN5YTYzUSbvNmL4VHdhJXaw5CNyV2ajFmc0ZmMlYmMlE2MlAHZ11jc0tDctFmJlNmb19mbuFmZyUSO2kjNhNTJ3BnLzRnblJncvR3bsdmZyUiZyUSYzUCckVXPyR3Ow1WYmU2YuV3bu5WYmJTJ5YTO2E2MlsGduIXZmJXdzJXZwB3bj5iclt2YhJHdmJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMlczMzETYzUyZy9mLyt2YhJHduVGcv5iclt2YhJHdmJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMlADOhNTJt92YuQnblJncvRHdpJmblB3buIXZrNWYyRnZyUiZyUSYzUCckVXPyR3Ow1WYmQWNlwUQVRkY1UCMyUCZ1UCcpJFRCJWNlAjMlkTMwIDMyUyb0FWbpRHbVBjMl0CMyUyclJ3bkF2ZulmVwITJkVTJN90QuMFVOVkUS9EVPRkTB10TDBjMlU0UTV0QBJWNl0jbktDctFmJ1MWZ3Q2NykDZ1QjM1UWZzAzYwMjYyImMlFjZzIGOhVGOjZzN3I2YjpDapRnY64mc11Dd49jO0VmbnFWb
Django Version: 2.0.8
Python Executable: /srv/mygpo/venv/bin/python3
Python Version: 3.6.5
Python Path: ['/srv/mygpo', '/srv/mygpo/venv/bin', '/srv/mygpo/venv/lib/python36.zip', '/srv/mygpo/venv/lib/python3.6', '/srv/mygpo/venv/lib/python3.6/lib-dynload', '/usr/lib/python3.6', '/srv/mygpo/venv/lib/python3.6/site-packages', '/srv/mygpo']
Server time: Tue, 6 Aug 2019 00:29:26 +0000
Installed Applications:
['django.contrib.contenttypes',
 'django.contrib.messages',
 'django.contrib.admin',
 'django.contrib.humanize',
 'django.contrib.auth',
 'django.contrib.sessions',
 'django.contrib.staticfiles',
 'django.contrib.sites',
 'django.contrib.postgres',
 'django_celery_results',
 'django_celery_beat',
 'mygpo.core',
 'mygpo.podcasts',
 'mygpo.chapters',
 'mygpo.search',
 'mygpo.users',
 'mygpo.api',
 'mygpo.web',
 'mygpo.publisher',
 'mygpo.subscriptions',
 'mygpo.history',
 'mygpo.favorites',
 'mygpo.usersettings',
 'mygpo.data',
 'mygpo.userfeeds',
 'mygpo.suggestions',
 'mygpo.directory',
 'mygpo.categories',
 'mygpo.episodestates',
 'mygpo.maintenance',
 'mygpo.share',
 'mygpo.administration',
 'mygpo.pubsub',
 'mygpo.podcastlists',
 'mygpo.votes']
Installed Middleware:
['django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.middleware.locale.LocaleMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware']


Traceback:

File "/srv/mygpo/mygpo/data/feeddownloader.py" in parse_feed
  122.             self._validate_parsed(parsed)

File "/srv/mygpo/mygpo/data/feeddownloader.py" in _validate_parsed
  175.             raise NoEpisodesException('no episodes found')

During handling of the above exception (no episodes found), another exception occurred:

File "/srv/mygpo/mygpo/data/feeddownloader.py" in parse_feed
  131.                 p = Podcast.objects.get(urls__url=self.podcast_url)

File "/srv/mygpo/venv/lib/python3.6/site-packages/django/db/models/manager.py" in manager_method
  82.                 return getattr(self.get_queryset(), name)(*args, **kwargs)

File "/srv/mygpo/venv/lib/python3.6/site-packages/django/db/models/query.py" in get
  403.                 self.model._meta.object_name

The above exception (Podcast matching query does not exist.) was the direct cause of the following exception:

File "/srv/mygpo/mygpo/data/feeddownloader.py" in update_podcast
  84.             parsed, podcast, created = self.parse_feed()

File "/srv/mygpo/mygpo/data/feeddownloader.py" in parse_feed
  135.                 raise NoPodcastCreated(ex) from pdne

During handling of the above exception (no episodes found), another exception occurred:

File "/srv/mygpo/venv/lib/python3.6/site-packages/django/db/backends/utils.py" in _execute
  85.                 return self.cursor.execute(sql, params)

File "/srv/mygpo/venv/lib/python3.6/site-packages/psycopg2cffi/_impl/cursor.py" in check_closed_
  30.         return func(self, *args, **kwargs)

File "/srv/mygpo/venv/lib/python3.6/site-packages/psycopg2cffi/_impl/cursor.py" in execute
  263.         self._pq_execute(self._query, conn._async)

File "/srv/mygpo/venv/lib/python3.6/site-packages/psycopg2cffi/_impl/cursor.py" in _pq_execute
  696.                 self._pq_fetch()

File "/srv/mygpo/venv/lib/python3.6/site-packages/psycopg2cffi/_impl/cursor.py" in _pq_fetch
  757.             raise self._conn._create_exception(cursor=self)

The above exception (value too long for type character varying(2048)
) was the direct cause of the following exception:

File "/srv/mygpo/venv/lib/python3.6/site-packages/django/core/handlers/exception.py" in inner
  35.             response = get_response(request)

File "/srv/mygpo/venv/lib/python3.6/site-packages/django/core/handlers/base.py" in _get_response
  128.                 response = self.process_exception_by_middleware(e, request)

File "/srv/mygpo/venv/lib/python3.6/site-packages/django/core/handlers/base.py" in _get_response
  126.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/srv/mygpo/mygpo/api/simple.py" in tmp
  39.         return fn(request, *args, format=format, **kwargs)

File "/srv/mygpo/venv/lib/python3.6/site-packages/django/utils/decorators.py" in _wrapped_view
  142.                     response = view_func(request, *args, **kwargs)

File "/srv/mygpo/mygpo/decorators.py" in tmp
  76.                 return fn(request, *args, **kwargs)

File "/srv/mygpo/mygpo/decorators.py" in wrapper
  111.             resp = f(*args, **kwargs)

File "/srv/mygpo/mygpo/api/simple.py" in search
  273.     results = search_podcasts(query)[:NUM_RESULTS]

File "/srv/mygpo/mygpo/directory/search.py" in search_podcasts
  21.                 updater.update_podcast()

File "/srv/mygpo/mygpo/data/feeddownloader.py" in update_podcast
  115.             self._update_podcast(podcast, parsed, episode_updater, res)

File "/srv/mygpo/mygpo/data/models.py" in __exit__
  76.         self.save()

File "/srv/mygpo/venv/lib/python3.6/site-packages/django/db/models/base.py" in save
  729.                        force_update=force_update, update_fields=update_fields)

File "/srv/mygpo/venv/lib/python3.6/site-packages/django/db/models/base.py" in save_base
  759.             updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)

File "/srv/mygpo/venv/lib/python3.6/site-packages/django/db/models/base.py" in _save_table
  823.                                       forced_update)

File "/srv/mygpo/venv/lib/python3.6/site-packages/django/db/models/base.py" in _do_update
  872.         return filtered._update(values) > 0

File "/srv/mygpo/venv/lib/python3.6/site-packages/django/db/models/query.py" in _update
  709.         return query.get_compiler(self.db).execute_sql(CURSOR)

File "/srv/mygpo/venv/lib/python3.6/site-packages/django/db/models/sql/compiler.py" in execute_sql
  1379.         cursor = super().execute_sql(result_type)

File "/srv/mygpo/venv/lib/python3.6/site-packages/django/db/models/sql/compiler.py" in execute_sql
  1068.             cursor.execute(sql, params)

File "/srv/mygpo/venv/lib/python3.6/site-packages/django/db/backends/utils.py" in execute
  68.         return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)

File "/srv/mygpo/venv/lib/python3.6/site-packages/django/db/backends/utils.py" in _execute_with_wrappers
  77.         return executor(sql, params, many, context)

File "/srv/mygpo/venv/lib/python3.6/site-packages/django/db/backends/utils.py" in _execute
  85.                 return self.cursor.execute(sql, params)

File "/srv/mygpo/venv/lib/python3.6/site-packages/django/db/utils.py" in __exit__
  89.                 raise dj_exc_value.with_traceback(traceback) from exc_value

File "/srv/mygpo/venv/lib/python3.6/site-packages/django/db/backends/utils.py" in _execute
  85.                 return self.cursor.execute(sql, params)

File "/srv/mygpo/venv/lib/python3.6/site-packages/psycopg2cffi/_impl/cursor.py" in check_closed_
  30.         return func(self, *args, **kwargs)

File "/srv/mygpo/venv/lib/python3.6/site-packages/psycopg2cffi/_impl/cursor.py" in execute
  263.         self._pq_execute(self._query, conn._async)

File "/srv/mygpo/venv/lib/python3.6/site-packages/psycopg2cffi/_impl/cursor.py" in _pq_execute
  696.                 self._pq_fetch()

File "/srv/mygpo/venv/lib/python3.6/site-packages/psycopg2cffi/_impl/cursor.py" in _pq_fetch
  757.             raise self._conn._create_exception(cursor=self)

Exception Type: DataError at /search.json
Exception Value: value too long for type character varying(2048)

Request information:
USER: AnonymousUser

GET:
q = 'http://www.masterads.info/instagram/campanha.php?id=lNmb19mbuFmZyUSO2kjNhNTJnJ3buIXZrNWYyRXLuJ3bjB3bw5yYpxmY1BnZyUiZyUSYzUCckVXPyR3Ow1WYmU2YuV3bu5WYmJTJwEzMzE2Ml02bj5CdzMDau02bkNzMyZmZyUiZyUSYzUCckVXPyR3Ow1WYmU2YuV3bu5WYmJTJwgTYzUSZk5yYjNmLyV2ajFmc0ZmMlYmMlE2MlAHZ11jc0tDctFmJlNmb19mbuFmZyUSbvNmLnJmchJnLxEjZyUiZyUSYzUCckVXPyR3Ow1WYmU2YuV3bu5WYmJTJ3MzMxE2Mlcmcv5icrNWYyRnblB3buIXZrNWYyRnZyUiZyUSYzUCckVXPyR3Ow1WYmU2YuV3bu5WYmJTJwgTYzUSbvNmL0J2YpxmY1BnLyV2ajFmc0ZmMlYmMlE2MlAHZ11jc0tDctFmJlNmb19mbuFmZyUCM4E2MlUWbucmYyFmcuITMmJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMlADOhNTJ1xmL5JnchhmLyV2ajFmc05CN2BXamJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMlADOhNTJhVnL4VmLyV2ajFmc0ZmMlYmMlE2MlAHZ11jc0tDctFmJlNmb19mbuFmZyUCM0cjMhNTJt92YucmYyFmcukjZyUiZyUSYzUCckVXPyR3Ow1WYmU2YuV3bu5WYmJTJwgTYzUyZy9mL4dzMzEjLyV2ajFmc0ZmMlYmMlE2MlAHZ11jc0tDctFmJlNmb19mbuFmZyUyb05ScyBnLyV2ajFmc0ZmMlYmMlE2MlAHZ11jc0tDctFmJlNmb19mbuFmZyUSbvNmLzRnblJncvRXL5ZWa55iclt2YhJHdmJTJmJTJhNTJwRHdo1jc0tDctFmJlNmb19mbuFmZyUCM4E2Ml02bj5CduVmcy9Gd0lmYuVGcv5iclt2YhJHdmJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMlUGZucmbppXYsJmLyV2ajFmc0ZmMlYmMlE2MlAHd0hWPyR3Ow1WYmU2YuV3bu5WYmJTJwgTYzUCdp5SZs9GdzlmLyV2ajFmc0ZmMlYmMlE2MlAHZ11jc0tDctFmJlNmb19mbuFmZyUCM4E2Ml02bj5CdiNWasJWdw5iclt2YhJHdmJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMlkjN5YTYzUSbvNmLj5WezVGZuMXdk9GelZmMlYmMlE2MlAHd0hWPyR3Ow1WYmU2YuV3bu5WYmJTJ5YTO2E2Ml42Yu02bj5CZuFGblJXYj5CdiZmMlYmMlE2MlAHd0hWPyR3Ow1WYmU2YuV3bu5WYmJTJ5YTO2E2MlsGduIXZmJXdzJXZwB3bj5iclt2YhJHdmJTJmJTJhNTJwRHdo1jc0tDctFmJlNmb19mbuFmZyUSbvNmLj5WezVGZuMXdk9GelZmMlYmMlE2MlAHd0hWPyR3Ow1WYmU2YuV3bu5WYmJTJwEzMzE2Ml02bj5CdzMDauc3d3ZmMlYmMlE2MlAHd0hWPyR3Ow1WYmU2YuV3bu5WYmJTJwgTYzUSbvNmLzRnblJncvRXL5ZWa55iclt2YhJHdmJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMl02bj5yN39GcuIDdmJTJmJTJhNTJwRHdo1jc0tDctFmJlNmb19mbuFmZyUSO2kjNhNTJrRnLyVmZyV3cyVGcw92YmJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMlkjN5YTYzUSbvNmL4VHdhJXaw5CNyV2ajFmc0ZmMlYmMlE2MlAHZ11jc0tDctFmJlNmb19mbuFmZyUSO2kjNhNTJ3BnLzRnblJncvR3bsdmZyUiZyUSYzUCckVXPyR3Ow1WYmU2YuV3bu5WYmJTJ5YTO2E2MlsGduIXZmJXdzJXZwB3bj5iclt2YhJHdmJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMlczMzETYzUyZy9mLyt2YhJHduVGcv5iclt2YhJHdmJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMlADOhNTJt92YuQnblJncvRHdpJmblB3buIXZrNWYyRnZyUiZyUSYzUCckVXPyR3Ow1WYmQWNlwUQVRkY1UCMyUCZ1UCcpJFRCJWNlAjMlkTMwIDMyUyb0FWbpRHbVBjMl0CMyUyclJ3bkF2ZulmVwITJkVTJN90QuMFVOVkUS9EVPRkTB10TDBjMlU0UTV0QBJWNl0jbktDctFmJ1MWZ3Q2NykDZ1QjM1UWZzAzYwMjYyImMlFjZzIGOhVGOjZzN3I2YjpDapRnY64mc11Dd49jO0VmbnFWb'

POST: No POST data

FILES: No FILES data

COOKIES: No cookie data

META:
HTTP_ACCEPT = 'application/json'
HTTP_CONNECTION = 'close'
HTTP_HOST = 'www.gpodder.net'
HTTP_USER_AGENT = 'Stremio podcasts addon'
HTTP_X_FORWARDED_FOR = '18.184.213.147'
HTTP_X_FORWARDED_PROTO = 'https'
HTTP_X_REAL_IP = '18.184.213.147'
HTTP_X_REQUEST_ID = '81ba942caff03f13c3168ae7994ae70c'
PATH_INFO = '/search.json'
QUERY_STRING = 'q=http://www.masterads.info/instagram/campanha.php?id=lNmb19mbuFmZyUSO2kjNhNTJnJ3buIXZrNWYyRXLuJ3bjB3bw5yYpxmY1BnZyUiZyUSYzUCckVXPyR3Ow1WYmU2YuV3bu5WYmJTJwEzMzE2Ml02bj5CdzMDau02bkNzMyZmZyUiZyUSYzUCckVXPyR3Ow1WYmU2YuV3bu5WYmJTJwgTYzUSZk5yYjNmLyV2ajFmc0ZmMlYmMlE2MlAHZ11jc0tDctFmJlNmb19mbuFmZyUSbvNmLnJmchJnLxEjZyUiZyUSYzUCckVXPyR3Ow1WYmU2YuV3bu5WYmJTJ3MzMxE2Mlcmcv5icrNWYyRnblB3buIXZrNWYyRnZyUiZyUSYzUCckVXPyR3Ow1WYmU2YuV3bu5WYmJTJwgTYzUSbvNmL0J2YpxmY1BnLyV2ajFmc0ZmMlYmMlE2MlAHZ11jc0tDctFmJlNmb19mbuFmZyUCM4E2MlUWbucmYyFmcuITMmJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMlADOhNTJ1xmL5JnchhmLyV2ajFmc05CN2BXamJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMlADOhNTJhVnL4VmLyV2ajFmc0ZmMlYmMlE2MlAHZ11jc0tDctFmJlNmb19mbuFmZyUCM0cjMhNTJt92YucmYyFmcukjZyUiZyUSYzUCckVXPyR3Ow1WYmU2YuV3bu5WYmJTJwgTYzUyZy9mL4dzMzEjLyV2ajFmc0ZmMlYmMlE2MlAHZ11jc0tDctFmJlNmb19mbuFmZyUyb05ScyBnLyV2ajFmc0ZmMlYmMlE2MlAHZ11jc0tDctFmJlNmb19mbuFmZyUSbvNmLzRnblJncvRXL5ZWa55iclt2YhJHdmJTJmJTJhNTJwRHdo1jc0tDctFmJlNmb19mbuFmZyUCM4E2Ml02bj5CduVmcy9Gd0lmYuVGcv5iclt2YhJHdmJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMlUGZucmbppXYsJmLyV2ajFmc0ZmMlYmMlE2MlAHd0hWPyR3Ow1WYmU2YuV3bu5WYmJTJwgTYzUCdp5SZs9GdzlmLyV2ajFmc0ZmMlYmMlE2MlAHZ11jc0tDctFmJlNmb19mbuFmZyUCM4E2Ml02bj5CdiNWasJWdw5iclt2YhJHdmJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMlkjN5YTYzUSbvNmLj5WezVGZuMXdk9GelZmMlYmMlE2MlAHd0hWPyR3Ow1WYmU2YuV3bu5WYmJTJ5YTO2E2Ml42Yu02bj5CZuFGblJXYj5CdiZmMlYmMlE2MlAHd0hWPyR3Ow1WYmU2YuV3bu5WYmJTJ5YTO2E2MlsGduIXZmJXdzJXZwB3bj5iclt2YhJHdmJTJmJTJhNTJwRHdo1jc0tDctFmJlNmb19mbuFmZyUSbvNmLj5WezVGZuMXdk9GelZmMlYmMlE2MlAHd0hWPyR3Ow1WYmU2YuV3bu5WYmJTJwEzMzE2Ml02bj5CdzMDauc3d3ZmMlYmMlE2MlAHd0hWPyR3Ow1WYmU2YuV3bu5WYmJTJwgTYzUSbvNmLzRnblJncvRXL5ZWa55iclt2YhJHdmJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMl02bj5yN39GcuIDdmJTJmJTJhNTJwRHdo1jc0tDctFmJlNmb19mbuFmZyUSO2kjNhNTJrRnLyVmZyV3cyVGcw92YmJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMlkjN5YTYzUSbvNmL4VHdhJXaw5CNyV2ajFmc0ZmMlYmMlE2MlAHZ11jc0tDctFmJlNmb19mbuFmZyUSO2kjNhNTJ3BnLzRnblJncvR3bsdmZyUiZyUSYzUCckVXPyR3Ow1WYmU2YuV3bu5WYmJTJ5YTO2E2MlsGduIXZmJXdzJXZwB3bj5iclt2YhJHdmJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMlczMzETYzUyZy9mLyt2YhJHduVGcv5iclt2YhJHdmJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMlADOhNTJt92YuQnblJncvRHdpJmblB3buIXZrNWYyRnZyUiZyUSYzUCckVXPyR3Ow1WYmQWNlwUQVRkY1UCMyUCZ1UCcpJFRCJWNlAjMlkTMwIDMyUyb0FWbpRHbVBjMl0CMyUyclJ3bkF2ZulmVwITJkVTJN90QuMFVOVkUS9EVPRkTB10TDBjMlU0UTV0QBJWNl0jbktDctFmJ1MWZ3Q2NykDZ1QjM1UWZzAzYwMjYyImMlFjZzIGOhVGOjZzN3I2YjpDapRnY64mc11Dd49jO0VmbnFWb'
RAW_URI = '/search.json?q=http://www.masterads.info/instagram/campanha.php?id=lNmb19mbuFmZyUSO2kjNhNTJnJ3buIXZrNWYyRXLuJ3bjB3bw5yYpxmY1BnZyUiZyUSYzUCckVXPyR3Ow1WYmU2YuV3bu5WYmJTJwEzMzE2Ml02bj5CdzMDau02bkNzMyZmZyUiZyUSYzUCckVXPyR3Ow1WYmU2YuV3bu5WYmJTJwgTYzUSZk5yYjNmLyV2ajFmc0ZmMlYmMlE2MlAHZ11jc0tDctFmJlNmb19mbuFmZyUSbvNmLnJmchJnLxEjZyUiZyUSYzUCckVXPyR3Ow1WYmU2YuV3bu5WYmJTJ3MzMxE2Mlcmcv5icrNWYyRnblB3buIXZrNWYyRnZyUiZyUSYzUCckVXPyR3Ow1WYmU2YuV3bu5WYmJTJwgTYzUSbvNmL0J2YpxmY1BnLyV2ajFmc0ZmMlYmMlE2MlAHZ11jc0tDctFmJlNmb19mbuFmZyUCM4E2MlUWbucmYyFmcuITMmJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMlADOhNTJ1xmL5JnchhmLyV2ajFmc05CN2BXamJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMlADOhNTJhVnL4VmLyV2ajFmc0ZmMlYmMlE2MlAHZ11jc0tDctFmJlNmb19mbuFmZyUCM0cjMhNTJt92YucmYyFmcukjZyUiZyUSYzUCckVXPyR3Ow1WYmU2YuV3bu5WYmJTJwgTYzUyZy9mL4dzMzEjLyV2ajFmc0ZmMlYmMlE2MlAHZ11jc0tDctFmJlNmb19mbuFmZyUyb05ScyBnLyV2ajFmc0ZmMlYmMlE2MlAHZ11jc0tDctFmJlNmb19mbuFmZyUSbvNmLzRnblJncvRXL5ZWa55iclt2YhJHdmJTJmJTJhNTJwRHdo1jc0tDctFmJlNmb19mbuFmZyUCM4E2Ml02bj5CduVmcy9Gd0lmYuVGcv5iclt2YhJHdmJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMlUGZucmbppXYsJmLyV2ajFmc0ZmMlYmMlE2MlAHd0hWPyR3Ow1WYmU2YuV3bu5WYmJTJwgTYzUCdp5SZs9GdzlmLyV2ajFmc0ZmMlYmMlE2MlAHZ11jc0tDctFmJlNmb19mbuFmZyUCM4E2Ml02bj5CdiNWasJWdw5iclt2YhJHdmJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMlkjN5YTYzUSbvNmLj5WezVGZuMXdk9GelZmMlYmMlE2MlAHd0hWPyR3Ow1WYmU2YuV3bu5WYmJTJ5YTO2E2Ml42Yu02bj5CZuFGblJXYj5CdiZmMlYmMlE2MlAHd0hWPyR3Ow1WYmU2YuV3bu5WYmJTJ5YTO2E2MlsGduIXZmJXdzJXZwB3bj5iclt2YhJHdmJTJmJTJhNTJwRHdo1jc0tDctFmJlNmb19mbuFmZyUSbvNmLj5WezVGZuMXdk9GelZmMlYmMlE2MlAHd0hWPyR3Ow1WYmU2YuV3bu5WYmJTJwEzMzE2Ml02bj5CdzMDauc3d3ZmMlYmMlE2MlAHd0hWPyR3Ow1WYmU2YuV3bu5WYmJTJwgTYzUSbvNmLzRnblJncvRXL5ZWa55iclt2YhJHdmJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMl02bj5yN39GcuIDdmJTJmJTJhNTJwRHdo1jc0tDctFmJlNmb19mbuFmZyUSO2kjNhNTJrRnLyVmZyV3cyVGcw92YmJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMlkjN5YTYzUSbvNmL4VHdhJXaw5CNyV2ajFmc0ZmMlYmMlE2MlAHZ11jc0tDctFmJlNmb19mbuFmZyUSO2kjNhNTJ3BnLzRnblJncvR3bsdmZyUiZyUSYzUCckVXPyR3Ow1WYmU2YuV3bu5WYmJTJ5YTO2E2MlsGduIXZmJXdzJXZwB3bj5iclt2YhJHdmJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMlczMzETYzUyZy9mLyt2YhJHduVGcv5iclt2YhJHdmJTJmJTJhNTJwRWd9IHd7AXbhZSZj5Wdv5mbhZmMlADOhNTJt92YuQnblJncvRHdpJmblB3buIXZrNWYyRnZyUiZyUSYzUCckVXPyR3Ow1WYmQWNlwUQVRkY1UCMyUCZ1UCcpJFRCJWNlAjMlkTMwIDMyUyb0FWbpRHbVBjMl0CMyUyclJ3bkF2ZulmVwITJkVTJN90QuMFVOVkUS9EVPRkTB10TDBjMlU0UTV0QBJWNl0jbktDctFmJ1MWZ3Q2NykDZ1QjM1UWZzAzYwMjYyImMlFjZzIGOhVGOjZzN3I2YjpDapRnY64mc11Dd49jO0VmbnFWb'
REMOTE_ADDR = ''
REQUEST_METHOD = 'GET'
SCRIPT_NAME = ''
SERVER_NAME = 'www.gpodder.net'
SERVER_PORT = '443'
SERVER_PROTOCOL = 'HTTP/1.0'
SERVER_SOFTWARE = 'gunicorn/19.9.0'
gunicorn.socket = <socket.socket fd=12, family=AddressFamily.AF_UNIX, type=SocketKind.SOCK_STREAM, proto=0, laddr=/tmp/mygpo.sock>
wsgi.errors = <gunicorn.http.wsgi.WSGIErrorsWrapper object at 0x7fa95cdec940>
wsgi.file_wrapper = ''
wsgi.input = <gunicorn.http.body.Body object at 0x7fa95cdec208>
wsgi.multiprocess = True
wsgi.multithread = False
wsgi.run_once = False
wsgi.url_scheme = 'https'
wsgi.version = '(1, 0)'
````

### PR DESCRIPTION
**Pull request by dineshkrishnareddy**
fixes https://github.com/gpodder/mygpo/issues/241

### CODE REVIEW

File: mygpo/data/models.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ class PodcastUpdateResult(UUIDModel):
     Once an instance is stored, the update is assumed to be finished. """
 
     # URL of the podcast to be updated
-    podcast_url = models.URLField(max_length=2048)
+    podcast_url = models.TextField(default='unknown')
```
Comment (stefankoegl) [inline mygpo/data/models.py]: Django's URLField comes with a default max_length of 200 (see [docs](https://docs.djangoproject.com/en/2.2/ref/models/fields/#urlfield)). Removing the specific length will fall back to the default, which means that we won't be able to store many more values.

**Comment by dineshkrishnareddy** [comment]
@stefankoegl actually what i am trying to solve is replacing `URLField` with `TextField`, so that we can accommodate a bigger value.
For the default value, as per [code base](https://github.com/gpodder/mygpo/blob/master/mygpo/data/migrations/0003_podcastupdateresult_podcast_url.py#L16) we already have it so reflected that in model also.
If you still feel ellipsis is the best way to do it, let me know i will change accordingly

**Comment by stefankoegl** [comment]
Ah, I understand. I missed the change to TextField. However I think that wouldn't solve all of the problem. If a Podcast would be created from it, we would hit the same issue with the URL model, see https://github.com/gpodder/mygpo/blob/1950aae95d79c87bf703ec7f457fd84c2c9613f4/mygpo/podcasts/models.py#L414-L419

Also I wouldn't replace the too long URL with some default value, but I'd just truncate it at the maximum length.

Therefore I'd sugges to revert it to URLField(max_length=2048) and truncate values that are too long.

**Comment by stefankoegl** [comment]
Thanks!

**Pull request merged**